JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R

```diff
@@ -230,27 +230,28 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
 
 .bainIndependentSamplesDescriptivesTable <- function(dataset, options, jaspResults, ready) {
 
-  if(!is.null(jaspResults[["descriptivesTable"]])) return() #The options for this table didn't change so we don't need to rebuild it
-		if(options[["descriptives"]]){
+  if (!is.null(jaspResults[["descriptivesTable"]])) return() #The options for this table didn't change so we don't need to rebuild it
+		if (options[["descriptives"]]) {
 
-	  descriptivesTable                                            <- createJaspTable("Descriptive Statistics")
-	  jaspResults[["descriptivesTable"]]                           <- descriptivesTable
+	  descriptivesTable <- createJaspTable("Descriptive Statistics")
 	  descriptivesTable$dependOn(options =c("variables", "descriptives", "descriptivesPlotsCredibleInterval", "groupingVariable"))
 		descriptivesTable$position <- 2
 
 	  descriptivesTable$addColumnInfo(name="v",                    title = "", type="string")
 		descriptivesTable$addColumnInfo(name="group",                title = "Group", type="string")
 	  descriptivesTable$addColumnInfo(name="N",                    title = "N", type="integer")
-	  descriptivesTable$addColumnInfo(name="mean",                 title = "Mean", type="number", format="sf:4;dp:3")
-	  descriptivesTable$addColumnInfo(name="sd",                   title = "sd", type="number", format="sf:4;dp:3")
-	  descriptivesTable$addColumnInfo(name="se",                   title = "se", type="number", format="sf:4;dp:3")
+	  descriptivesTable$addColumnInfo(name="mean",                 title = "Mean", type="number")
```

Comment:
Am I correct that we do not need to formatting in general anymore, because formatting is controlled by jaspResults?

Reply:
`format="sf:4;dp:3"` is standard for number, so if you don't want anything fancy specifying `type="number"` is sufficient.